confgenerator/testdata/invalid/linux/metrics-receiver_invalid_parameter_malformed_collection_interval/golden_error

```diff
@@ -1,9 +1 @@
-[5:28] "collection_interval" must be a duration of at least 10s
-   2 |   receivers:
-   3 |     receiver_1:
-   4 |       type: hostmetrics
->  5 |       collection_interval: 24
-                                  ^
-   6 |   service:
-   7 |     pipelines:
-   8 |       default_pipeline:
\ No newline at end of file
+cannot unmarshal uint64 into Go struct field UnifiedConfig.Metrics of type time.Duration
```

Comment:
[Optional] Sigh. This is a net negative for usability, but there's precedent for messages like this, so we can probably kick that can down the road and address all of those in one fell swoop at some point…

Reply:
Yeah, fixing this generically for all fields would be a great separate PR.